ADOPTERS.md

```diff
@@ -0,0 +1,37 @@
+# Adopters
+
+**📢 If you're using JSON Schema in your organization, please add your company name to this list.**
+**🙏 It really helps the project to gain momentum and credibility. It's a small contribution back to the project with a big impact.**
+
+Why add your company name?
+
+- Demonstrates the real-world usage of JSON Schema.
+- Provides visibility to organizations that have successfully adopted JSON Schema.
+- Encourages other organizations to adopt JSON Schema.
+- Builds trust in the JSON Schema ecosystem.
+
+This document also lists the organizations that use JSON Schema based on public information available in blog posts, events, and videos. If any organization would like to be added or get removed please make a Pull Request.
+
+Guidelines:
+- Please don't include your organization's logo or other trademarked material
+- Add a reference (link to a public blog post, video, slides, etc) mentioning that JSON Schema is used
+- Please include a contact link for maintainers
+
+Looking for extra credit? We also do [case studies](https://github.com/orgs/json-schema-org/projects/8/views/2) and accept sponsorship via [Open Collective](https://opencollective.com/json-schema).
```

Comment:
I think it'd be slightly prettier to link to the subset of the blog rather than a project board (i.e. published ones rather than project management).

Reply:
I totally agree... but currently there's no such link or filter 😅

Reply:
Oh bizarre. As in the next.js thing takes post types (which I know we set to Case Study) but does nothing with that (i.e. doesn't generate pages per-type??) Whatever I won't think too hard, I know this is changing soon.